packages/smooth_app/lib/generic_lib/widgets/smooth_text_form_field.dart

```diff
@@ -30,12 +32,18 @@ class SmoothTextFormField extends StatefulWidget {
   final bool? enabled;
   final TextInputAction? textInputAction;
   final String? Function(String?)? validator;
+
+  /// takes in a function to run onFieldSubmit action
+  final void Function(String?)? onSubmit;
```

Comment:
Have added these for documentation purpose since I saw a github workflow to generate dart documentation via `dart doc .` command to generate documentation hence I added the documentation for the work added by me. Let me know if I need to remove them.